how-to/how-to-use-virtio-mem-with-kata.md

```diff
@@ -0,0 +1,49 @@
+# Kata Containers with virtio-mem
+
+- [Introduction](#introduction)
+- [Pre-requisites](#pre-requisites)
+- [Run a Kata Container with utilizing virtio-mem](#run-a-kata-container-utilizing-virtio-mem)
+
+## Introduction
+
+The basic idea of virtio-mem is to provide a flexible, cross-architecture memory hot(un)plug solution that avoids many limitations imposed by existing technologies, architectures, and interfaces. More details can be found in https://lkml.org/lkml/2019/12/12/681.
+
+Kata Containers with virtio-mem support memory resize.
+
+## Pre-requisites
+
+Kata Containers with virtio-mem requires Linux and the qemu that support virtio-mem.
+
+The Linux kernel is at https://github.com/davidhildenbrand/linux/tree/virtio-mem-rfc-v4.
+The Linux kernel config that can work with Kata Containers is at https://gist.github.com/teawater/016194ee84748c768745a163d08b0fb9.
+
+The qemu is at https://github.com/teawater/qemu/tree/kata-virtio-mem. (The original source is at https://github.com/davidhildenbrand/qemu/tree/virtio-mem.  Its base version of qemu cannot work with Kata Containers.  So merge the commit of virtio-mem to upstream qemu.)
+
+Set Linux and the qemu that support virtio-mem with following line in the Kata Containers configuration.
+```
+path = "qemu-dir"
+kernel = "vmlinux-dir"
+```
+
+Enable virtio-mem with following line in the Kata Containers configuration.
```

Comment:
Can you mention the section under which this configuration exists in the configuration.toml for completeness?

Reply:
Changed this part to:
```
Set Linux and the QEMU that support `virtio-mem` with following line in the Kata Containers QEMU configuration configuration-qemu.toml.
```